### ISSUE TITLE
Upload multiple items spreadsheet items not added to exhibit

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/exhibits#1910]
Using the Curation > Add items > Upload multiple items spreadsheet in this exhibit: https://exhibits.stanford.edu/test-stn/resources/new

I filled out the spreadsheet for four records, using the URL, title, and date fields. 

I uploaded the spreadsheet and got the expected flash message, and a few minutes later got the expected email telling me the four items had been indexed.

However, 20 minutes later I still don't see these four items on the Curation > Items page, and they aren't returned when I do a "find_all" search.

The spreadsheet I used is here: https://drive.google.com/file/d/1gFRCkwJggzIocc3F_ArT3QnYAatVxEdF/view?usp=sharing

**Comment by jkeck** [comment]
@ggeisler right off the bat, it looks like the image links (at least one of them that I checked) isn't actually a link to the image, it's a link to an HTML page that happens to display an image, but it's not the image itself.

We don't (and really cannot) parse arbitrary HTML pages like those that host images (e.g. Google, Box, etc) and "pluck" images out of that HTML page for uploading.

Can you re-test this using URLs to where the indexer can find the source images themselves?

**Comment by mejackreed** [comment]
@ggeisler you may be able to derive this using:

url: `https://drive.google.com/file/d/15KcFmztipv9ZwAqQz7CuZximtNiMwqpg/view?usp=sharing` where `id = 15KcFmztipv9ZwAqQz7CuZximtNiMwqpg`

to =>
`https://lh3.googleusercontent.com/d/{id}`

**Comment by ggeisler** [comment]
Thanks @jkeck I think you're correct about the links. I just saw the big image on the page and assumed the link was good, but I see what you mean about it not actually a link to the image itself.

I figured out how to use direct links (similar to what @mejackreed just suggested, but a different thing where I had to substitute `uc?export=view&id=` for part of the drive URL) and the spreadsheet upload does work (yay!).

<img width="703" alt="Screen Shot 2020-08-05 at 6 40 52 PM" src="https://user-images.githubusercontent.com/101482/89480924-709bd000-d74b-11ea-8c75-1f14409b5eaa.png">

There's still the problem with the missing images, but I'm sure that's the same issue as https://github.com/sul-dlss/exhibits/issues/1903. So I'll close this. Thanks @jkeck and @mejackreed.
